whisper.py

```diff
@@ -57,15 +60,25 @@ def run_asr(input_path, output_dir, model=None) -> dict | str:
         if not model:
             logger.info("Model not passed as param, need to obtain it first")
             model = load_model(model_base_dir, w_model, w_device)
+        if w_device == "cpu":
+            logger.warning(f"Device selected is {w_device}: using a batch size of 1")
+
+        os.environ["PYTORCH_KERNEL_CACHE_PATH"] = model_base_dir
         logger.info("Processing segments")
+
+        if w_device == "cuda":
+            gpu_mem_measure = GpuMemoryMeasure()
+            gpu_mem_measure.start_measure_gpu_mem()
+
         segments, _ = model.transcribe(
             input_path,
             vad_filter=w_vad,
             beam_size=w_beam_size,
             best_of=w_best_of,
-            # temperature=ast.literal_eval(w_temperature),
-            language="nl",
+            batch_size=w_batch_size if w_device == "cuda" else 1,
+            language="nl",  # TODO: experiment without language parameter specified (for programs with foreign speech)
             word_timestamps=w_word_timestamps,
+            log_progress=True,
```

Comment:
Just not sure whether a progress bar would be very useful in our setup

Reply:
Agreed, and from my testing it seems a bit too technical (it shows the number of batches, but no duration or anything that's easier to understand as a human)